applications/navi/doc/README.md

```diff
@@ -0,0 +1,88 @@
+# Overview
+The Navi application is used for listening to notifications through amqp and using them to create push notifications for iOS and Android devices.
+
+## Architectural Overview
+Navi functionality is implemented across several modules:
+* cb\_push\_notification\_subscriptions
+* navi\_listener
+* navi\_module\_sup
+* navi\_notification\_server\_sup
+* nv\_apns
+* nv\_fcm
+* kapi\_navi
```

Comment:
AMQP APIs that are shared among apps should go in `core/kazoo_amqp/api` and be named generically, like `kapi_websockets` instead of `kapi_blackhole` even though `blackhole` does the websocket interactions.

If the intent is for other kazoo applications (like crossbar) to use the `kapi_navi` payloads, `kapi_navi` should move to core and be renamed to `kapi_push_notifications` or the like.